This is a testharness.js-based test.
[FAIL] IDB respects bucket quota
  promise_rejects_quotaexceedederror: function "function() { throw e; }" threw object "QuotaExceededError" that is not a correct QuotaExceededError: property "code" is equal to 0, expected 22
Harness: the test ran to completion.

